src/azure-cli/azure/cli/command_modules/acs/_validators.py

```diff
@@ -686,3 +686,33 @@ def validate_os_sku(namespace):
             CONST_OS_SKU_CBLMARINER,
             CONST_OS_SKU_MARINER,
         )
+
+
+def validate_utc_offset(namespace):
+    """Validates --utc-offset for aks maintenanceconfiguration add/update commands."""
+    if namespace.utc_offset is None:
+        return
+    utc_offset_regex = re.compile(r'^[+-]\d{2}:\d{2}$')
```

Comment:
Please add comments to these regular expressions and add examples to the comments to increase the readability of the code

Reply:
Thank you for pointing this out, I've added comments for clarity.